src/storage-controller/src/instance.rs

```diff
@@ -411,12 +473,17 @@ where
     ///
     /// If `send_commands` is true, will send commands for newly-scheduled
```

Comment:
This doc comment only references ingestions, it would be nice to update it to all talk about exports. Specifically I'm thinking a description of how a Kafka ingestion is multi-replica, but a Kafka export is single-replica

Reply:
Good catch, wlll fix